examples/demo.css

```diff
@@ -201,6 +201,19 @@ h1 {
   margin-bottom: 20px;
 }
 
+
+.savesequencemsg{
```

Comment:
Ah, do you think we could opt for a standard Bootstrap "alert" style, to avoid the custom styling here? I like it but perhaps it'll be more efficient to use a pre-existing one! https://getbootstrap.com/docs/3.3/ Thanks!!!

Reply:
Sure! I will do that. Thanks!